lisa/core/nodeFactory.py

```diff
@@ -17,7 +19,9 @@ def createNodeFromConfig(id: str, config: Dict[str, object]) -> Optional[Node]:
             constants.ENVIRONMENTS_NODES_REMOTE,
         ]:
             is_default = cast(bool, config.get(constants.IS_DEFAULT, False))
-            node = Node.createNode(id, node_type=node_type, isDefault=is_default)
+            node = Node.createNode(
```

Comment:
is it formatted by 88 len rule? I have auto format enabled, it may be changed back and forth by our rules...

Reply:
It got longer because I renamed `id` to `identifier` so as to not shadow the built-in Python function `id` (which also confused the type checker).